src/main/java/amidst/gui/profileselect/LocalProfileComponent.java

```diff
@@ -81,6 +82,13 @@ private boolean tryLoad() {
 		try {
 			Log.i("using minecraft launcher profile '" + getProfileName() + "' with versionId '" + getVersionName()
 					+ "'");
+					
+			String possibleModProfiles = "optifine|forge";
+            if (Pattern.matches(possibleModProfiles, getProfileName().toLowerCase())) {
```

Comment:
One more thing: Please don't check against the profile name, but against the version name. The profile name is chosen by the user and can thus be something like "1.8.9-without-optifine". The current approach would block this profile. Using the version name will prevent this issue.